pkg/api/aim/dashboards.go

```diff
@@ -179,6 +179,7 @@ func UpdateDashboard(c *fiber.Ctx) error {
 	}
 
 	if err := database.DB.
+		Omit(clause.Associations).
```

Comment:
this is because you select too many fields above — we have no need for `Type` or `IsArchived` from the `App`

Reply:
reverted back for now.